tensorflow_recommenders_addons/dynamic_embedding/core/ops/hkv_hashtable_ops.cc

```diff
@@ -308,6 +308,7 @@ REGISTER_OP(PREFIX_OP_NAME(HkvHashTableOfTensors))
     .Attr("max_hbm_for_vectors: int = 0")
     .Attr("step_per_epoch: int = 0")
     .Attr("strategy: int = 0")
+    .Attr("reserved_key_start_bit: int = 0")
```

Comment:
default value of reserved_key_start_bit is better to be same as python code which is 1.